<?xml version="1.0" encoding="UTF-8"?><record_update table="sys_metadata_link">
    <sys_metadata_link action="INSERT_OR_UPDATE">
        <directory>update</directory>
        <documentkey>46e2fee9a9fe19810049b49dee0daf58</documentkey>
        <payload><![CDATA[<?xml version="1.0" encoding="UTF-8"?><record_update sys_domain="global" table="incident">
    <incident action="INSERT_OR_UPDATE">
        <active>true</active>
        <activity_due>2022-09-16 21:50:33</activity_due>
        <additional_assignee_list/>
        <approval>not requested</approval>
        <approval_history/>
        <approval_set/>
        <assigned_to display_value="Don Goodliffe">9ee1b13dc6112271007f9d0efdb69cd0</assigned_to>
        <assignment_group display_value="Software" name="Software">8a4dde73c6112278017a6a4baf547aa7</assignment_group>
        <business_duration>1970-01-06 00:00:00</business_duration>
        <business_impact/>
        <business_service/>
        <business_stc>432000</business_stc>
        <calendar_duration>1970-01-06 00:00:24</calendar_duration>
        <calendar_stc>432024</calendar_stc>
        <caller_id display_value="Fred Luddy">5137153cc611227c000bbd1bd8cd2005</caller_id>
        <category>software</category>
        <cause/>
        <caused_by/>
        <child_incidents/>
        <close_code/>
        <close_notes/>
        <closed_at>2022-06-07 23:39:10</closed_at>
        <closed_by/>
        <cmdb_ci display_value="Saints and Sinners Bingo">46c7318aa9fe198100c76003f0bc82e9</cmdb_ci>
        <comments/>
        <comments_and_work_notes/>
        <company display_value="ACME North America">31bea3d53790200044e0bfc8bcbe5dec</company>
        <contact_type>phone</contact_type>
        <contract/>
        <correlation_display/>
        <correlation_id/>
        <delivery_plan/>
        <delivery_task/>
        <description>There was a software update two days ago and now my VPN doesn't work. Nothing loads when I click on the VPN client icon on my desktop.</description>
        <due_date/>
        <escalation>0</escalation>
        <expected_start/>
        <follow_up/>
        <group_list/>
        <hold_reason/>
        <impact>1</impact>
        <incident_state>2</incident_state>
        <knowledge>false</knowledge>
        <location display_value="324 South State Street, Salt Lake City,UT">105cf7f3c611227501e75e08b14a38ba</location>
        <made_sla>false</made_sla>
        <notify>1</notify>
        <number>INC0000015</number>
        <opened_at>2022-06-02 23:38:46</opened_at>
        <opened_by display_value="ITIL User">681b365ec0a80164000fb0b05854a0cd</opened_by>
        <order/>
        <origin_id/>
        <origin_table/>
        <parent/>
        <parent_incident/>
        <priority>1</priority>
        <problem_id/>
        <reassignment_count>2</reassignment_count>
        <rejection_goto/>
        <reopen_count/>
        <reopened_by/>
        <reopened_time/>
        <resolved_at/>
        <resolved_by/>
        <rfc/>
        <route_reason/>
        <service_offering/>
        <severity>3</severity>
        <short_description>I can't launch my VPN client since the last software update</short_description>
        <sla_due>2022-06-22 01:00:00</sla_due>
        <state>2</state>
        <subcategory/>
        <sys_class_name>incident</sys_class_name>
        <sys_created_by>admin</sys_created_by>
        <sys_created_on>2022-06-18 23:40:15</sys_created_on>
        <sys_domain>global</sys_domain>
        <sys_domain_path>/</sys_domain_path>
        <sys_id>46e2fee9a9fe19810049b49dee0daf58</sys_id>
        <sys_mod_count>11</sys_mod_count>
        <sys_updated_by>admin</sys_updated_by>
        <sys_updated_on>2022-09-16 19:50:33</sys_updated_on>
        <task_effective_number>INC0000015</task_effective_number>
        <time_worked/>
        <universal_request/>
        <upon_approval/>
        <upon_reject/>
        <urgency>1</urgency>
        <user_input/>
        <variables/>
        <watch_list/>
        <wf_activity/>
        <work_end/>
        <work_notes/>
        <work_notes_list/>
        <work_start/>
        <x_921453_test_incident_tests/>
        <x_921453_test_test_short_description_length>59</x_921453_test_test_short_description_length>
    </incident>
</record_update>
]]></payload>
        <sys_class_name>sys_metadata_link</sys_class_name>
        <sys_created_by>admin</sys_created_by>
        <sys_created_on>2022-12-09 05:24:23</sys_created_on>
        <sys_id>f9fd1a571be319107221877ddc4bcb5e</sys_id>
        <sys_name>INC0000015</sys_name>
        <sys_package display_value="test" source="x_921453_test">2b9a8cd38d221110d5125b5353465bd9</sys_package>
        <sys_policy/>
        <sys_scope display_value="test">2b9a8cd38d221110d5125b5353465bd9</sys_scope>
        <sys_update_name>sys_metadata_link_f9fd1a571be319107221877ddc4bcb5e</sys_update_name>
        <tablename>incident</tablename>
    </sys_metadata_link>
</record_update>
